src/main/java/com/synopsys/integration/alert/web/api/user/UserController.java

```diff
@@ -51,76 +55,81 @@
     public static final String USER_BASE_PATH = ConfigController.CONFIGURATION_PATH + "/user";
 
     private final Logger logger = LoggerFactory.getLogger(UserController.class);
-    private final ContentConverter contentConverter;
-    private final ResponseFactory responseFactory;
     private final AuthorizationManager authorizationManager;
     private final UserActions userActions;
     private final UserManagementDescriptorKey descriptorKey;
 
     @Autowired
-    public UserController(ContentConverter contentConverter, ResponseFactory responseFactory, AuthorizationManager authorizationManager, UserActions userActions,
+    public UserController(AuthorizationManager authorizationManager, UserActions userActions,
         UserManagementDescriptorKey descriptorKey) {
-        this.contentConverter = contentConverter;
-        this.responseFactory = responseFactory;
         this.authorizationManager = authorizationManager;
         this.userActions = userActions;
         this.descriptorKey = descriptorKey;
     }
 
     @GetMapping
-    public ResponseEntity<String> getAllUsers() {
+    public List<UserConfig> getAllUsers() {
```

Comment:
Can we wrap this List in an object? 
For exmaple:
```java
public class MultiUserConfigResponseModel {
    private List<UserConfig> users;
    .
    .
    .
}
```

Reply:
Sure, something similar to MultiRolePermissionModel.java that just has methods to get the UserConfig list?